include/bout/index_derivs_interface.hxx

```diff
@@ -237,10 +239,11 @@ T D4DY4(const T& f, CELL_LOC outloc = CELL_DEFAULT, const std::string& method =
     return standardDerivative<T, DIRECTION::YOrthogonal, DERIV::StandardFourth>(
         f, outloc, method, region);
   } else {
-    const T f_aligned = toFieldAligned(f, "RGN_NOX");
+    const bool is_unaligned = (f.getDirectionY() == YDirectionType::Aligned);
```

Comment:
Think this should be `YDirectionType::Standard`

Reply:
Thanks, this was a particularly stupid typo from when I couldn't make up my mind which way round to do the condition!